File: CS/ListViewCountInTabs.Module/BusinessObjects/Task.cs
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;

namespace ListViewCountInTabs.Module.BusinessObjects
{
    [DefaultClassOptions]
    public class Task : BaseObject
    {
        public Task(Session session) : base(session) { }

        string subject;
        public string Subject
        {
            get { return subject; }
            set { SetPropertyValue(nameof(Subject), ref subject, value); }
        }

        Employee assignedTo;
        [Association("Employee-Tasks")]
        public Employee AssignedTo
        {
            get { return assignedTo; }
            set { SetPropertyValue(nameof(AssignedTo), ref assignedTo, value); }
        }
    }
}

File: DetailViewTabCount/Module/VB/DetailViewTabCountVB.Module/BusinessObjects/Task.vb
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo

<DefaultClassOptions>
Public Class Task
	Inherits BaseObject

	Public Sub New(ByVal session As Session)
		MyBase.New(session)
	End Sub

	Private _subject As String
	Public Property Subject() As String
		Get
			Return _subject
		End Get
		Set(ByVal value As String)
			SetPropertyValue(NameOf(Subject), _subject, value)
		End Set
	End Property

	Private _assignedTo As Employee
	<Association("Employee-Tasks")>
	Public Property AssignedTo() As Employee
		Get
			Return _assignedTo
		End Get
		Set(ByVal value As Employee)
			SetPropertyValue(NameOf(AssignedTo), _assignedTo, value)
		End Set
	End Property
End Class
